3.processing_features/scripts/0.merge_sc_cytotable.py

```diff
@@ -118,19 +123,40 @@
 
 for plate, info in plate_info_dictionary.items():
     file_path = pathlib.Path(info["dest_path"])
-    
+
     # Load the DataFrame from the Parquet file
     df = pd.read_parquet(file_path)
 
-    # If any, drop rows where "Metadata_ImageNumber" is NaN (artifact of cytotable)
-    df = df.dropna(subset=["Metadata_ImageNumber"])
-
-    # Columns to move to the front
-    columns_to_move = ['Nuclei_Location_Center_X', 'Nuclei_Location_Center_Y', 'Cells_Location_Center_X', 'Cells_Location_Center_Y']
+    # Update Image_Metadata_Plate column if plate is Plate_6
+    # TODO: Remove this once the metadata is fixed in the CellProfiler pipeline
```

Comment:
Remove todo?

Reply:
Not yet! I will remove it once the plate is reran and I will make a PR for it! :)